src/resolve/resolved-link.c

```diff
@@ -816,7 +816,8 @@ ResolveSupport link_get_mdns_support(Link *link) {
         return MIN(link->mdns_support, link->manager->mdns_support);
 }
 
-int link_address_new(Link *l, LinkAddress **ret, int family, const union in_addr_union *in_addr) {
+int link_address_new(Link *l, LinkAddress **ret, int family, const union in_addr_union *in_addr, const union
+                in_addr_union *broadcast) {
```

Comment:
see CODING_STYLE

we usually would indent this either in one line, and if it grows too long then like this:

```c
int link_address_new(
                Link *l, 
                LinkAddress **ret, 
                int family, 
                const union in_addr_union *in_addr, 
                const union in_addr_union *broadcast) {
```

(i.e. that's double the regular indentation → 16ch, and then one argument per line)

Reply:
Gotcha